contracts/tests/the-explorer-guild-mint/claim.test.ts

```diff
@@ -103,16 +103,16 @@ Clarinet.test({
       ),
     ]);
 
-    for (let index = 0; index < maxMintNumber; index++) {
+    for (let index = 0; index < maxMintNumber / 10; index++) {
       block = chain.mineBlock([
         Tx.contractCall(
           "the-explorer-guild-mint",
-          "claim",
+          "claim-ten",
```

Comment:
Had to optimise this test, looks like the new version of clarity is failing to run it and becomes responsive after some time.